src/core/meshcop/dtls.cpp

```diff
@@ -333,33 +332,28 @@ void Dtls::SetSslAuthMode(bool aVerifyPeerCertificate)
 
 otError Dtls::Stop(void)
 {
+    VerifyOrExit((mState == kStateConnecting) || (mState == kStateConnected));
```

Comment:
Since the two states are checked a lot. Is it possible to define stopped=0 connecting=1 connected=3 closenotify=2, so that we can define an inline method isactive() and check the two states by mState&0x1?